admin/openwisp-config/Makefile

```diff
@@ -1,14 +1,14 @@
-# openwisp.org
+# SPDX-Identifier-License: GPL-3.0-only
 #
-# This is free software, licensed under the GNU General Public License v3.
-# See /LICENSE for more information.
+# openwisp.org
```

Comment:
IMHO this should be done treewide not only for these two packages. Both approaches are OK for me.

Reply:
Can you be a bit more specific? Not sure about which change you are talking.

Reply:
Your's change related to SPDX License Identifier. Now, you are making it more confusing, at least from my side. Because some packages have variant A, some of them have your variant. Thus it should be done for all packages at the same time.

Reply:
Yeah.